server/logging/CoreLogback.java

```diff
@@ -50,19 +50,26 @@ public static void configure(LoggerContext logContext, Config.Log logConfig) {
         encoder.setLayout(layout);
 
         Map<String, Appender<ILoggingEvent>> appenders = new HashMap<>();
-        logConfig.output().outputs().forEach((name, outputType) -> {
-            if (outputType.isStdout()) {
-                appenders.put(name, consoleAppender(name, logContext, encoder, layout));
-            } else if (outputType.isFile()) {
-                appenders.put(name, fileAppender(name, logContext, encoder, layout, outputType.asFile()));
-            } else throw TypeDBException.of(ILLEGAL_STATE);
-        });
+        logConfig.output().outputs().forEach((name, outputType) ->
+                appenders.put(name, appender(name, logContext, encoder, layout, outputType)));
 
         configureRootLogger(logConfig.logger().defaultLogger(), logContext, appenders);
         logConfig.logger().filteredLoggers().values().forEach(l -> configureLogger(l, logContext, appenders));
     }
 
-    private static void configureLogger(Config.Log.Logger.Filtered logger, LoggerContext context,
+    protected Appender<ILoggingEvent> appender(String name,
```

Comment:
Extract the appender creation function, sot that it can easily be extended by the subclasses.